manufacturing/filter_holder/README.md

```diff
@@ -30,26 +40,31 @@ BEFORE purchasing any parts.**
 
 **NOTE:** Before you buy anything on this list, check with Edwin, Martin or Ethan. Blower takes a while to deliver.
```

Comment:
Actually this stuff about blower is irrelevant. Maybe we can remove it? Only the note you added should be here.